dashboard/app/views/home/_gallery.html.haml

```diff
@@ -1,9 +1,30 @@
+-show_heading = local_assigns[:show_heading]
+
 .course_progress
-  .row#lesson
-    .span12#heading
-      %h3.whitetext{style: 'margin: 0'}
-        = t("home.your_gallery")
+  -if show_heading
+    .row#lesson
+      .span12#heading
+        %h3.whitetext{style: 'margin: 0'}
+          = t("home.your_gallery")
   .row#stats
     .span12{style: 'margin-left: 10px;'}
       %div.gallery#gallery
         = render partial: 'home/gallery_content'
+
+:javascript
+  $(document).ready(function() {
```

Comment:
I've fixed the indentation in the inline js. PTAL

In addition to the entry point for `home/index`, this js would also need to be included in the entry point for `projects/index` since we're showing the gallery there too. Alternatively, there are some partials such as `layouts/_terms_interstitial` or `home/_homepage` which require js directly while obeying the entry point rules. This alternative seems more straight forward and I'm happy to do it if you think it's a good idea.